### PR TITLE
Removes unreachable code

### PR DESCRIPTION
**Pull request by spookydonut**
## About The Pull Request
As detected by https://github.com/SpaceManiac/SpacemanDMM/pull/123

### CODE REVIEW

File: code/modules/mob/living/simple_animal/friendly/drone/interaction.dm [hunk 1]
```diff
@@ -115,8 +115,8 @@
 		return
 	if(hack)
 		if(hacked)
-			return
 			Stun(40)
```
Comment (AnturK) [inline code/modules/mob/living/simple_animal/friendly/drone/interaction.dm]: This should probably just be unindented.

Comment (spookydonut) [reply code/modules/mob/living/simple_animal/friendly/drone/interaction.dm]: I'm unsure what the intended behavior is here so I'm just guessing

File: code/modules/clothing/under/miscellaneous.dm [hunk 2]
```diff
@@ -127,7 +127,6 @@
 			to_chat(user, "<span class='notice'>You refill the suit's built-in extinguisher, using up the cartridge.</span>")
 			qdel(E)
 			return
-		return
 	return
```
Comment (Cyberboss) [inline code/modules/clothing/under/miscellaneous.dm]: Remove all the return calls in this proc please

File: code/modules/modular_computers/file_system/programs/arcade.dm [hunk 3]
```diff
@@ -77,7 +77,6 @@
 		pause_state = FALSE
 		game_check()
 		return
```
Comment (Cyberboss) [inline code/modules/modular_computers/file_system/programs/arcade.dm]: Remove these 3 as well please

**Review by Cyberboss — CHANGES_REQUESTED** [review]

File: code/modules/modular_computers/file_system/programs/arcade.dm (hunk 3 above)
Comment (Cyberboss) [reply code/modules/modular_computers/file_system/programs/arcade.dm]: The ones in the if/else if/else chain

File: code/modules/mob/living/simple_animal/friendly/drone/interaction.dm (hunk 1 above)
Comment (ninjanomnom) [reply code/modules/mob/living/simple_animal/friendly/drone/interaction.dm]: The hacked check looks like it's supposed to be an early return and the stun is for when they first get hacked

**Pull request merged**